components/mpas-seaice/bld/build-namelist

```diff
@@ -841,6 +841,7 @@ add_default($nl, 'config_new_snow_density');
 add_default($nl, 'config_max_snow_density');
 add_default($nl, 'config_minimum_wind_compaction');
 add_default($nl, 'config_wind_compaction_factor');
+add_default($nl, 'config_wind_snow_loss_factor');
```

Comment:
I think this should be called `config_wind_snow_redistribution_factor` (or leave out wind if this is too long) because the factor sets the ratio of snow depth on level ice versus ridged ice. The percentage of snow lost into leads depends on this factor as well as the areas of level and ridged ice and open water.  To reduce confusion, I think the factor's name should only reference the redistribution to prevent someone from thinking that 30% (or whatever fraction) of the snow is lost into leads -- that's not what's happening.